pandas/core/groupby/groupby.py

```diff
@@ -581,6 +582,17 @@ def wrapper(*args, **kwargs):
             if 'axis' not in kwargs_with_axis or \
                kwargs_with_axis['axis'] is None:
                 kwargs_with_axis['axis'] = self.axis
+            if (name == 'hist' and
+                    kwargs.get('equal_bins', False) is True):
+                # GH-22222
+                bins = kwargs.get('bins')
+                if bins is None:
+                    bins = 10  # use default value used in `hist_series`
+                if is_integer(bins):
```

Comment:
Could just use `is_scalar` here since we only need to guard against scalar vs sequence